pkg/codegen/pcl/resource.go

```diff
@@ -43,6 +43,8 @@ type ResourceOptions struct {
 	Version model.Expression
 	// The plugin download URL for this resource.
 	PluginDownloadURL model.Expression
+	// The resource whose deletion will cause the deletion of this resource.
```

Comment:
This comment doesn't sound quite right... I think it should be more like the other doc comments, e.g.:

```suggestion
	// If set, the provider's Delete method will not be called for this resource if the specified resource is being
	// deleted as well.
```